### ISSUE TITLE
Should 'luminosity' be included in contrast discussion?

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-quick-start#36]
In tip: [Provide sufficient color contrast](http://w3c.github.io/wai-quick-start/designing.html#provide-sufficient-color-contrast)

Does the inclusion of the more correct term 'luminosity' add an unnecessary level of technical terminology?

**Comment by AndrewArch** [comment]
I think introducing the correct technical is probably appropriate for this audience.
BTW, maybe we can make the 'bad' example worse? (unless the point was to she that you often don't need to tweak much to meet the requirement :)

**Comment by iadawn** [comment]
@AndrewArch, in response to your second point; yes, the aim was to try to show that sometimes not a lot is needed.

**Comment by hkramer** [comment]
No, I don't think it's necessary to add the term "luminosity". What we have now is sufficient.

**Comment by slhenry** [comment]
I feel strongly that we should *not* use the phrase "color contrast"! It is incorrect. Refer to EOWG previous discussions on this for Easy Checks. Here's what we did there: Contrast ratio ("color contrast")... (This accessibility requirement is sometimes called sufficient "color contrast"; however, that is incorrect — technically it's "luminance contrast". On this page we use "contrast ratio" as short for "luminance contrast ratio" because it's less jargony.) <http://www.w3.org/WAI/eval/preliminary.html#contrast> 

I think something like "Provide sufficient contrast between colors" is fine for the tip text. I think it'd be good to use "contrast ratio" at least in the explanation since that has strong tie in to the WCAG requirement. I don't think we *need* to bring in "luminance"; however, we could include that statement like in Easy Checks.

Also: I actually think we want to downplay "sufficient" and that we might want to actually state 4.5:1. I think the write up in Easy Checks is good -- and some of that is important. I know we want to keep this short; however, I think we need to say more here &/or in the developing one -- e.g., "While some people need high contrast, for others,  bright colors (high luminance) are not readable. They need low luminance. ... Web pages should also have a minimum contrast by default: a contrast ratio of at least 4.5:1 for normal-size text. ... Web browsers should allow people to change the color of text and background, and web pages need to work when people change colors."

**Comment by sharronrush** [comment]
Agree that WAI should use technically accurate vocabulary and so we should refer to luminance and luminosity.

**Comment by bakkenb** [comment]
+1 to Shawn about using "contrast ratio" in language when possible. Also instead of "sufficient", we should be saying "meets minimum..."

I am okay with using luminosity in the explanation area if it fits nicely and can introduce a few designers to some of the technical vocabulary.

In the second sentence, strike "Don't worry about", and replace with something like "Does not apply to".

**Comment by schantzwebconsulting** [comment]
Yes, it should.

Second sentence should use "Logos and incidental text are exempt from... "

**Comment by iadawn** [comment]
Note, I feel really quite strongly about avoiding stating the actual ratio. Ideally, given the audience, I think the aim is to highlight the issue, and point them to a tool or list of tools that will allow them to determine if they have a problem. I think the ratio on it's own doesn't help that much and I know designers that would cry if they saw the equation to calculate the ratio.

Also, there is quite a bit in [Shawn's comment](https://github.com/w3c/wai-quick-start/issues/36#issuecomment-115268034) that speaks to why this is important. I recall that the group agreed to keep this out as much as possible.

**Comment by Egyrs** [comment]
I think it is important to use the right technology. At the end of the day people read these documents to learn and it is important that they get used to the terminology used properly.

**Comment by yatil** [comment]
I think it would be OK to just use “Use sufficient contrast”.

**Comment by iadawn** [comment]
[EOWG Meeting 26 Jun](http://www.w3.org/2015/06/26-eo-minutes.html#item01): Resolved to avoid the term ‘color contrast’ and introduce a note about ‘luminance contrast’
